src/index.js

```diff
@@ -166,15 +168,16 @@ class PubsubBaseProtocol extends EventEmitter {
    * @private
    * @param {Object} props
    * @param {string} props.protocol
-   * @param {DuplexStream} props.strean
+   * @param {Stream} props.stream
```

Comment:
```suggestion
   * @param {IterableStream} props.stream
```
Per https://gist.github.com/alanshaw/591dc7dd54e4f99338a347ef568d6ee9, we should call this an IterableStream. This is not a node stream and might create confusion

Reply:
I called it `DuplexIterableStream` so its known to be a duplex object (`{ sink, source }`)